client/state/sites/vouchers/reducer.js

```diff
@@ -28,6 +33,28 @@ import { itemsSchema } from './schema';
 export const items = ( state = {}, action ) => {
 	const { siteId } = action;
 	switch ( action.type ) {
+		case SITE_VOUCHERS_ASSIGN:
+			switch ( action.serviceType ) {
```

Comment:
Is this switch structure necessary? It looks like if we add other services here, they will all need the same code, only having `GOOGLE_AD_CREDITS` replaced with something else, like `OTHER_AD_CREDITS`. It seems like we could get rid of this whole level of `switch`/indentation and replace `GOOGLE_AD_CREDITS` in the following code with `action.serviceType`